src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -24,6 +24,8 @@
 {
   // This is the physical size of a single map square in pixels.
   public static final int baseSpriteSize = 16;
+  
+  public static final String DEFAULT_SPRITE_KEY = "DEFAULT";
```

Comment:
Maybe `DEFAULT_FACTION_NAME` would be more descriptive of how it's actually used?